code-reviews/recipes/javascript-and-typescript.md

```diff
@@ -118,15 +118,28 @@ By default, we use the following overrides should be added to the VS Code config
 
 ## Build Validation
 
-To automate this process in Azure Devops you can add the following snippet to you `azure-pipelines.yaml` file. This will lint any scripts in the `./scripts/` folder.
-
-> TODO: Add example of yaml for automating linting and testing
+To automate this process in Azure Devops you can add the following snippet to you pipeline definition yaml file. This will lint any scripts in the `./scripts/` folder.
```

Comment:
Typo: you -> your 

But how about: "As an example of automating linting in build validation process, the following snippet of an Azure DevOps pipeline definition YAML file will run a linter to inspect scripts in `./scripts` folder:"